app/views/providers/application_merits_task/domestic_abuse_summaries/show.html.erb

```diff
@@ -1,49 +1,41 @@
 <%= form_with(
       model: @form,
-      url: providers_legal_aid_application_opponent_path,
+      url: providers_legal_aid_application_domestic_abuse_summary_path,
       method: :patch,
       local: true
     ) do |form| %>
-    <%= page_template(
-            page_title: t('.h1-heading'),
-            back_link: {},
-            form: form) do %>
-
-    <%= form.govuk_text_field(
-          :full_name,
-          label: { text: t('.opponent_name'), size: 'm' },
-          class: 'govuk-input govuk-input--width-20'
-        ) %>
-
-    <div class="govuk-!-padding-bottom-2"></div>
+  <%= page_template(
+        page_title: t('.h1-heading'),
+        back_link: {},
+        form: form) do %>
 
     <%
-      questions = %i[understands_terms_of_court_order warning_letter_sent].map do |attr|
+      questions = %i[warning_letter_sent].map do |attr|
         OpenStruct.new(
           title: t(".#{attr}"),
           attribute: attr,
           attribute_details: "#{attr}_details".to_sym,
-        )
+          )
       end
       questions += %i[police_notified].map do |attr|
         OpenStruct.new(
-            title: t(".#{attr}"),
-            attribute: attr,
-            attribute_details_true: "#{attr}_details_true".to_sym,
-            attribute_details_false: "#{attr}_details_false".to_sym
-            )
+          title: t(".#{attr}"),
+          attribute: attr,
+          attribute_details_true: "#{attr}_details_true".to_sym,
+          attribute_details_false: "#{attr}_details_false".to_sym
+        )
       end
       questions += %i[bail_conditions_set].map do |attr|
         OpenStruct.new(
-            title: t(".#{attr}"),
-            attribute: attr,
-            attribute_details: "#{attr}_details".to_sym,
-            )
+          title: t(".#{attr}"),
+          attribute: attr,
+          attribute_details: "#{attr}_details".to_sym,
+          )
       end
       police_notified_titles = {
-          yes: t('.police_notified_yes'),
-          no: t('.police_notified_no'),
-        }
+        yes: t('.police_notified_yes'),
+        no: t('.police_notified_no'),
+      }
```

Comment:
**NOT a blocker**, and I guess order matters, but lines 14-18 and 29-33 appear to be the same so could possibly be combined with 
```
questions = %i[warning_letter_sent bail_conditions_set].map do |attr|
  OpenStruct.new(
    title: t(".#{attr}"),
    attribute: attr,
    attribute_details: "#{attr}_details".to_sym,
  )
end
```

OR if order matters could this be refactored to not use the `map`
e.g.

```
questions = OpenStruct.new(
              title: t(".warning_letter_sent"),
              attribute: :warning_letter_sent,
              attribute_details: :warning_letter_sent_details,
            )

questions += OpenStruct.new(
                title: t(".police_notified"),
                attribute: :police_notified,
                attribute_details_true: :police_notified_details_true,
                attribute_details_false: police_notified_details_true,
              )
...
```

or 🤔, could use form to store a helper in some way. And use Struct instead as OpenStruct is slow


``` 
# DomesticeAbuseSummaryForm

QuestionStruct = Struct.new(
                title:,
                attribute:,
                attribute_details:,
                attribute_details_true:,
                attribute_details_false:,
              )


  def warning_letter_sent
    QuestionStruct.new(
      title: t(".warning_letter_sent"), 
      attribute: :warning_letter_sent,
      attribute_details: :warning_letter_sent_details,
    )
  end

# view
questions = [@form.warning_letter_sent, ...]
```


Anyway, just a refactor to extract pre-existing code from view so should be done after the fact to avoid a bigger PR

Reply:
I lifted and shifted this from the existing page, and removed some bits...
It's certainly not my preferred way of rending a form! 
I will drop a ticket in the backlog to investigate and refactor this view 👍